ntbs-service-tests/UnitTests/Services/NotificationServiceTest.cs

```diff
@@ -51,6 +51,34 @@ public void DeathDate_IsSetToNullIfPostMortemFalse()
             Assert.Null(timeline.DeathDate);
         }
 
+        [Fact]

```

Comment:
I tried to combine these two tests into a theory but got a very strange, seemingly unrelated error about user secrets and gave up :(